bokehjs/src/coffee/models/tools/inspectors/hover_tool.coffee

```diff
@@ -317,44 +318,48 @@ export class HoverTool extends InspectTool
       callback:     [ p.Any                    ] # TODO: p.Either(p.Instance(Callback), p.Function) ]
     }
 
-  initialize: (attrs, options) ->
-    super(attrs, options)
+  connect_signals: () ->
+    super()
+    # TODO: @connectTo(@plot.properties.renderers.change, () -> @_computed_renderers = @_ttmodels = null)
```

Comment:
I will address this in another PR, because this kind of listeners need more attention across bokehjs.